src/wizards.ts

```diff
@@ -0,0 +1,250 @@
+import { ListItem } from '@material/mwc-list/mwc-list-item';
+import { html } from 'lit-element';
+import { repeat } from 'lit-html/directives/repeat';
+import { get, translate } from 'lit-translate';
+
+import {
+  EditorAction,
+  identity,
+  isEqual,
+  isSame,
+  newWizardEvent,
+  SimpleAction,
+  Wizard,
+  WizardAction,
+} from './foundation.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195001)
```suggestion
} from "./foundation.js";
```